sample-app/src/pages/LocationsPage.tsx

```diff
@@ -0,0 +1,72 @@
+import ResultsCount from '../components/ResultsCount';
+import AlternativeVerticals from '../components/AlternativeVerticals';
+import AppliedFilters from '../components/AppliedFilters';
+import DirectAnswer from '../components/DirectAnswer';
+import VerticalResults from '../components/VerticalResults';
+import SpellCheck from '../components/SpellCheck';
+import LocationBias from '../components/LocationBias';
+import { StandardCard } from '../components/cards/StandardCard';
+import useAnswersOnPage from '../hooks/useAnswersOnPage';
+import Facets from '../components/Facets';
+import FilterSearch from '../components/FilterSearch';
+
+const filterSearchFields = [{
+  fieldApiName: 'name',
+  entityType: 'location'
+}, {
+  fieldApiName: 'paymentOptions',
+  entityType: 'location'
+}, {
+  fieldApiName: 'services',
+  entityType: 'location'
+}];
+
+export default function LocationsPage({ verticalKey }: {
+  verticalKey: string
+}) {
+  useAnswersOnPage(verticalKey);
```

Comment:
`useAnswersOnPage` makes me think the hook is returning some `answers` variable

Reply:
maybe the name can say something about seeding initial state or setting up the page? I was thinking `useAnswersPageSetup`

Reply:
or even something like `usePageSetupEffect`? I know I'm bringing up an already resolved comment but I still think the name could be tweaked a little

Reply:
Sure, i'll update to `usePageSetupEffect`